corehq/apps/hqwebapp/decorators.py

```diff
@@ -236,6 +237,22 @@ def _inner(request, *args, **kwargs):
     return _inner
 
 
+def setup_two_factor(view_func):
+    """
+    Use this decorator on the dispatch method of a TemplateView subclass
+
+    We use custom forms for two factor setup and authentication
+    This decorate should be use on any view that depends on two factor
```

Comment:
```suggestion
    This decorator should be use on any view that depends on two factor
```

Reply:
No longer relevant as this decorator was removed in 4ccbb9f9c74ee16ae5179cd22299b53e1dbdcabf